pkgs/development/web/nodejs/nodejs.nix

```
@@ -129,7 +127,24 @@ let
 
     pos = builtins.unsafeGetAttrPos "version" args;
 
-    inherit patches;
+    patches = patches ++ [
+      (fetchpatch2 {
+        name = "add-qemu-wrapper-support.patch";
+        url = "https://github.com/buildroot/buildroot/raw/ecf060a2ff93626fb5fbe1c52b8d42b6ee0fbb5b/package/nodejs/nodejs-src/0001-add-qemu-wrapper-support.patch";
```

GitHub repository is a mirror, the official Git repository is at https://gitlab.com/buildroot.org/buildroot.
```suggestion
        url = "https://gitlab.com/buildroot.org/buildroot/-/raw/ecf060a2ff93626fb5fbe1c52b8d42b6ee0fbb5b/package/nodejs/nodejs-src/0001-add-qemu-wrapper-support.patch";
```